src/unix/pty.cc

```diff
@@ -203,7 +203,11 @@ NAN_METHOD(PtyFork) {
 
   // termios
   struct termios* term = new termios();
+#if defined(IUTF8)
   term->c_iflag = ICRNL | IXON | IXANY | IMAXBEL | BRKINT | IUTF8;
+#else
```

Comment:
This is definitely an improvement, I think I got confused looking at the individual commits and thought you were removing functionality (you were but from your own commit). Thanks @jbenden this will make it into the next version.